--- live555ProxyServer.cpp	2021-01-09 19:31:51.000000000 +0300
+++ live555ProxyServer.OutPacketBuffer.cpp	2021-01-10 11:28:20.000000000 +0300
@@ -58,7 +58,7 @@
 int main(int argc, char** argv) {
   // Increase the maximum size of video frames that we can 'proxy' without truncation.
   // (Such frames are unreasonably large; the back-end servers should really not be sending frames this large!)
-  OutPacketBuffer::maxSize = 100000; // bytes
+  OutPacketBuffer::maxSize = 168780; // bytes
 
   // Begin by setting up our usage environment:
   TaskScheduler* scheduler = BasicTaskScheduler::createNew();
